capstone/src/main/webapp/script.js

```diff
@@ -6,3 +6,48 @@ var instance = M.Tabs.init(elem, options);
 //Initialize sidenav
 const slide_menu = document.querySelectorAll(".sidenav");
 M.Sidenav.init(slide_menu, {});
+
+//Get Search Request From User and display result
+async function studySetsBySearch() {
+  const stringToSearchBy = document.getElementById("search").value;
+  const response = await fetch(
+    "/study_set?stringToSearchBy=" + stringToSearchBy
+  );
+  const searchResponse = await response.json();
+  var configuredSetsToPost = configureCardsForSearchPage(searchResponse);
+  document.getElementById("results-container").innerHTML = configuredSetsToPost;
+}
+
+function configureCardsForSearchPage(studySets) {
+  var configuredCardStudySets = "";
+  var counter = 0;
+
+  for (var studySet of studySets) {
+    configuredCardStudySets += configureCard(studySet);
+    counter++;
+  }
+
+  if (counter == 0) {
+    return (configuredCardStudySets = ` <div class="no-results"><h3>No Results Found...<i class="large material-icons">sentiment_dissatisfied</i></h3></div>`);
+  }
+  return configuredCardStudySets;
+}
+
+function configureCard(studySet) {
```

Comment:
See previous comment: Rename this method to make it more obvious that this method returns HTML for a given card